docs/source/tutorials/getting_started/compile.rst

```diff
@@ -46,6 +46,11 @@ In particular, OpenFPGA requires specific versions for the following dependencie
 
 :iverilog:
   version 10.1+ is required to run Verilog-to-Verification flow
+  
+:envyaml:
```

Comment:
@ganeshgore  Need your input here. Is it possible to install python packages from the [requirements.txt](https://github.com/lnis-uofu/OpenFPGA/blob/master/requirements.txt) file?
If so, we can simple talk about how to use the requirements.txt file rather showing low-level details.